apis/pubsub/v1alpha1/generate.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}/dev/tools/controllerbuilder
+
+
+go run . generate-types \
+    --service google.pubsub.v1 \
+    --api-version pubsub.cnrm.cloud.google.com/v1alpha1 \
+    --resource PubSubSnapshot:Snapshot
+
+go run . generate-mapper \
+    --service google.pubsub.v1 \
+    --api-version pubsub.cnrm.cloud.google.com/v1alpha1
```

Comment:
This process helps us aligning the proto between api and fuzzer/mapper. 

The problem is that this [pubsub v1 is redirected to v2](https://pkg.go.dev/cloud.google.com/go/pubsub@v1.50.0/apiv1/pubsubpb) which causes the auto-mapper file (v2) mismatches the other controller dir (v1).